### PR TITLE
Added jsdocs for several components

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**
- Help inline
- Text highlight
- Label
- Tokens
- Wait

#122

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-jsdocs](https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-jsdocs)

File: src/app/public/modules/text-highlight/text-highlight.directive.ts [hunk 1]
```diff
@@ -50,6 +63,10 @@ export class SkyTextHighlightDirective implements OnChanges, AfterViewInit, OnDe
     }
   }
 
+  /**
+   * Angular's ngOnDestroy lifecycle hook.
+   * @internal
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/text-highlight/text-highlight.directive.ts:68]: ☝️ Angular's lifecycle hooks are automatically removed from docs.

File: src/app/public/modules/tokens/token.component.ts [hunk 2]
```diff
@@ -27,6 +34,10 @@ export class SkyTokenComponent {
     return !!this._disabled;
   }
 
+  /**
+   * Sets the `aria-label` and title attributes for the token's close button to support accessibility.
+   * @default "Remove item"
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/tokens/token.component.ts:39]:
```suggestion
   * @default 'Remove item'
```

File: src/app/public/modules/tokens/tokens.component.ts [hunk 3]
```diff
@@ -66,6 +78,10 @@ export class SkyTokensComponent implements OnInit, OnChanges, OnDestroy {
     return (this._dismissible !== false);
   }
 
+  /**
+   * Specifies the token property to display for each item in the tokens list.
+   * @default "name"
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/tokens/tokens.component.ts:83]:
```suggestion
   * @default 'name'
```

File: src/app/public/modules/tokens/tokens.component.ts [hunk 4]
```diff
@@ -158,6 +213,10 @@ export class SkyTokensComponent implements OnInit, OnChanges, OnDestroy {
     }
   }
 
+  /**
+   * Angular's ngOnDestroy lifecycle hook.
+   * @internal
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/tokens/tokens.component.ts:218]: ☝️ Angular's lifecycle hooks are automatically removed from docs.

File: src/app/public/modules/tokens/tokens.component.ts [hunk 5]
```diff
@@ -208,6 +276,9 @@ export class SkyTokensComponent implements OnInit, OnChanges, OnDestroy {
     this.notifyTokenSelected(token);
   }
 
+  /**
+   * @internal
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/tokens/tokens.component.ts:280]: ☝️ The docs tools only exposes `Input` and `Output`; we don't need all these `@internal` comments.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-jsdocs](https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-jsdocs)

File: src/app/public/modules/tokens/tokens.component.ts (hunk 5 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/tokens/tokens.component.ts]: Removed.

File: src/app/public/modules/tokens/tokens.component.ts (hunk 4 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/tokens/tokens.component.ts]: Removed.

File: src/app/public/modules/tokens/tokens.component.ts (hunk 3 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/tokens/tokens.component.ts]: Updated.

File: src/app/public/modules/text-highlight/text-highlight.directive.ts (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/text-highlight/text-highlight.directive.ts]: Removed.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-jsdocs](https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-jsdocs)

**Pull request merged**